ql/src/java/org/apache/hadoop/hive/ql/metadata/HiveStorageHandler.java

```diff
@@ -280,6 +283,21 @@ default boolean supportsPartitionTransform() {
     return null;
   }
 
+  /**
+   * Creates a DynnamicPartitionCtx instance that will be set up by the storage handler itself. Useful for non-native
+   * tables where partitions are not handled by Hive, and sorting is required in a custom way before writing the table.
```

Comment:
What happens if the return value is `null`

Reply:
fixing javadoc